q2_feature_table/plugin_setup.py

```diff
@@ -174,14 +174,25 @@
                 "to define the mapping of IDs to a group."
 )
 
+i_table, p_overlap_method, o_table = TypeMap({
+    (FeatureTable[Frequency],
+     Str % Choices(sorted(q2_feature_table.overlap_methods()))):
+    FeatureTable[Frequency],
+    (FeatureTable[RelativeFrequency],
+     # We don't want to allow summing of RelativeFrequency tables, so remove
+     # that option from the overlap methods
+     Str % Choices(sorted(q2_feature_table.overlap_methods() - {'sum'}))):
```

Comment:
I made a minor tweak here by converting the `overlap_methods` to a set, that way we can clearly say which element is being removed here, rather than doing some indexing arithmetic in my head.

Reply:
LGTM, thanks @thermokarst